### DISCUSSION TITLE
Notification custom routing should be applicable for all orgs

### DISCUSSION THREAD

**Discussion by AverageMarcus** [community/community#32726] [Other Feature Feedback, Questions, & Ideas]
I use custom routing to make sure work-related and personal-related emails go to the appropriate email but this only covers so much.

When working on work-related activities on open source repos where I'm not a member of the org all notifications go to my default (personal) email. 

Ideally, the custom routing shouldn't be limited to just orgs I'm a member of but an unrestricted list of any org on GitHub.

Example:

| Source | Dest | Expected |
| --- | --- | --- |
| my-company-org | company email | Yes |
| AverageMarcus/* | personal email | Yes |
| kubernetes-sig | personal email | No |

In the above example, I do a lot of work on repos within the `kubernetes-sig` org but I'm not a member of that org. I'd like all notifications related to that org to be routable to my work email.

**Comment by svenssonaxel** [comment]
See #47242
